.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,31 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/go
+// If you're wondering how to get SSH working for git pushes, make sure to ssh-add ~/.ssh/<YOURPRIVATEKEY> on your host system
+// to make sure the ssh agent has your key
+{
+	"name": "Go",
+	// Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
+	"image": "mcr.microsoft.com/devcontainers/go:1-1.21-bullseye",
```

Comment:
Would it make sense to have the Go version string here be managed by [`updatecli`](https://www.updatecli.io/)? I think what that means specifically is that you'll want to add a new target in this section: https://github.com/elastic/beats/blob/3161fc0cd2e4250508ab4a40bd2debdf0883bdb7/.ci/bump-golang.yml#L86

Reply:
Yes! Would you be able to add that as a commit?

Reply:
What is the difference between `go:1-1.21-bullseye` and `go:1.21-bullseye`? I found https://github.com/devcontainers/images/tree/main/src/go#using-this-image but I couldn't understand the effect of adding or leaving out the extra `1-` at the start of the tag.

Reply:
> Yes! Would you be able to add that as a commit?

Done via 9a897f1f3796e06bc533c2e9151fbd62531438d2 and 29fdddf7a333eca9bdcb953e80212e3f95268c72.